src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -31,6 +92,45 @@ const ManageUsers: () => JSX.Element = () => {
           textAlign="center"
         >
           <Heading>Manage Users</Heading>
+          <Table>
+            <TableCaption placement="top">Current Admin Users</TableCaption>
+            <Thead>
+              <Tr>
+                <Th>Name</Th>
+                <Th>Email</Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {users
+                .filter(user => user.admin)
+                .map((user, id) => (
+                  <Tr key={id}>
+                    <Td>{user.name}</Td>
+                    <Td>{user.email}</Td>
+                  </Tr>
+                ))}
+            </Tbody>
+          </Table>
+          <Table>
+            <TableCaption placement="top">All Current Users</TableCaption>
+            <Thead>
+              <Tr>
+                <Th>Name</Th>
+                <Th>Email</Th>
+                <Th>Admin?</Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {users.map((user, id) => (
+                <Tr key={id}>
+                  <Td>{user.name}</Td>
+                  <Td>{user.email}</Td>
+                  <Td>{user.admin ? 'Yes' : 'No'}</Td>
+                </Tr>
+              ))}
+            </Tbody>
+          </Table>
+          {error && <Text textColor="red.500">{error}</Text>}
           <Button as="a" href="/admin" margin={1}>
```

Comment:
What's the difference between as vs on click redirect?

Reply:
I thought it was more readable this way and it means I didn't have to import the Router here, instead I just have an a element that's styled as a Button